spec/requests/as_content_owner/asset_spec.rb

```diff
@@ -4,13 +4,19 @@
 describe Yt::Asset, :partner do
   describe '.ownership' do
     let(:asset) { Yt::Asset.new id: asset_id, auth: $content_owner }
+    let(:asset_with_metadata_mine) { Yt::Asset.new id: asset_id, auth: $content_owner, params: {fetch_metadata: 'mine'} }
```

Comment:
the asset has to have metadataMine? how about give it another asset ID or pull out as different `describe`?

Reply:
Yes, the asset has to have params in order to fetch data. I can move into its own describe.

Reply:
what about a describe titled `'given an asset administered by the content owner that we want to fetch metadata for'`?

Reply:
I pulled it into a different describe.

Reply:
sorry @jcohenho I wasn't clear. I actually meant by another `describe` it doesn't look great if we put everything under `describe '.ownership'`. And I also thought we should be able to use a separate asset ID for `asset_with_metadata_mine` because not all asset has `metadata_mine` value.